scripts/main.bro

```diff
@@ -79,6 +83,8 @@ function effective_tld(domain: string): string
 function effective_domain(domain: string): string
 	{
 	local depth=2;
+	if ( effective_tlds_4th_level in domain )
+		depth=5;
 	if ( effective_tlds_3rd_level in domain )
```

Comment:
Whoops, there was one more.

```suggestion
	if ( effective_tlds_3rd_level in domain )
```

Reply:
I'm wondering if we're hitting the limits of "suggested changes" here. I had made a suggested change for line 88 to add else-if there as well. Then you appeared to have caught that change was needed independently, and suggested the same change. Now I'm seeing something different in your suggestion, but I think we are in sync as far as the actual branches go. Does it look like that to you?